packages/server-bridge/src/factories/callHandlersSeriallyFactory.js

```diff
@@ -0,0 +1,22 @@
+import Promise from 'bluebird';
+import size from 'lodash/size';
+
+function doCallHandlers(log, handlers, index, { session, socket }) {
```

Comment:
you could define doCallHandlers inside your factory function to dorp the first argument